package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "bedrock-payment-http",
+  "version": "0.0.1",
+  "description": "HTTP APIs for bedrock-payment.",
+  "main": "lib/index.js",
+  "directories": {
+    "lib": "lib"
+  },
+  "scripts": {
+    "test": "mocha"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/digitalbazaar/bedrock-payment-http.git"
+  },
+  "keywords": [
+    "HTTPS",
+    "API",
+    "Payment",
+    "Bedrock",
+    "Express",
+    "Javascript"
+  ],
+  "author": {
+    "name": "Digital Bazaar, Inc.",
+    "email": "support@digitalbazaar.com",
+    "url": "https://digitalbazaar.com"
+  },
+  "bugs": {
+    "url": "https://github.com/digitalbazaar/bedrock-payment-http/issues"
+  },
+  "homepage": "https://github.com/digitalbazaar/bedrock-payment-http#readme",
+  "devDependencies": {
+    "eslint": "^6.6.0",
+    "eslint-config-digitalbazaar": "^2.0.1",
+    "eslint-plugin-jsdoc": "^17.1.0"
+  },
+  "peerDependencies": {
+    "bedrock": "^2.0.0",
+    "bedrock-payment": "0.0.1",
+    "bedrock-passport": "^5.0.0",
+    "bedrock-validation": "^4.1.0",
+    "express-async-handler": "^1.1.4"
```

Comment:
this can be eliminated by using the export provided by bedrock-express.

Reply:
Addressed here: https://github.com/digitalbazaar/bedrock-payment-http/pull/1/commits/3dbb7eb8ce3be7e2d9e6d23ef1efbb2c6a1b31a1